Source/BugsnagSessionTrackingApiClient.m

```diff
@@ -19,8 +19,11 @@ - (NSOperation *)deliveryOperation {
 }
 
 - (void)deliverSessionsInStore:(BugsnagSessionFileStore *)store {
+    NSString *apiKey = self.config.apiKey;
+    NSURL *sessionURL = self.config.sessionURL;
```

Comment:
Both of these values should be [copied](https://developer.apple.com/documentation/objectivec/nsobject/1418807-copy?language=objc)